central/nodecomponentedge/datastore/datastore_sac_test.go

```diff
@@ -45,16 +42,14 @@ func (s *nodeComponentEdgeDatastoreSACTestSuite) SetupSuite() {
 	s.datastore, err = GetTestPostgresDataStore(s.T(), pool)
 	s.Require().NoError(err)
 	s.testContexts = sacTestUtils.GetNamespaceScopedTestContexts(context.Background(), s.T(), resources.Node)
+	err = s.testGraphDatastore.PushNodeToVulnerabilitiesGraph()
+	s.Require().NoError(err)
 }
 
 func (s *nodeComponentEdgeDatastoreSACTestSuite) TearDownSuite() {
```

Comment:
It might be worth having the deferred call to s.Require().NoError(s.testGraphDatastore.CleanNodeToVulnerabilitiesGraph()) as part of the test suite tear down.

Reply:
I thought about this but we will drop entire database so I think no more cleaning is needed

Reply:
fair enough :)